h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2176,6 +2298,256 @@ private void fitIRLSM(Solver s) {
+          predictorSizeChange =  !coeffNames.equals(Arrays.asList(_state.activeData().coefNames()));
+          if (predictorSizeChange) {  // reset if predictors changed
+            coeffNames = changeCoeffBetainfo(_state.activeData()._coefNames);
+            _state.resizeConstraintInfo(equalityConstraints, lessThanEqualToConstraints, coeffNames, betaCnd.length);
+            ginfo = gam.equals(_parms._glmType) ? new GLMGradientSolver(_job, _parms, _state.activeData(), 0,
+                    _state.activeBC(), _betaInfo, _penaltyMatrix, _gamColIndices) : new GLMGradientSolver(_job, _parms,
+                    _state.activeData(), 0, _state.activeBC(), _betaInfo);
+          }
+          // solve for GLM coefficients
+          betaCnd = constraintGLM_solve(gram);  // beta_k+1 = beta_k+dk where dk = beta_k+1-beta_k   
+          predictorSizeChange = !coeffNames.equals(Arrays.asList(_state.activeData().coefNames()));
+          if (predictorSizeChange) {  // reset if predictors changed
+            coeffNames = changeCoeffBetainfo(_state.activeData()._coefNames);
+            _state.resizeConstraintInfo(equalityConstraints, lessThanEqualToConstraints, coeffNames, betaCnd.length);
+            ginfo = gam.equals(_parms._glmType) ? new GLMGradientSolver(_job, _parms, _state.activeData(), 0,
+                    _state.activeBC(), _betaInfo, _penaltyMatrix, _gamColIndices) : new GLMGradientSolver(_job, _parms,
+                    _state.activeData(), 0, _state.activeBC(), _betaInfo);
+          }
+
+          if (applyBetaConstraints)
+            bc.applyAllBounds(betaCnd);
+
+          // add line search for GLM coefficients
+          if (ls == null)
+            ls = new ExactLineSearch(betaCnd, _state, coeffNames);
+          else
+            ls.reset(betaCnd, _state, coeffNames);
+
+          if (ls.findAlpha(lambdaEqual, lambdaLessThan, _state, equalityConstraints, lessThanEqualToConstraints, ginfo, firstIter) || firstIter) {
+            betaCnd = ls._newBeta;
+            gradientInfo = ls._ginfoOriginal;
+          } else {  // line search failed for some reason
+            Log.info(LogMsg("Ls failed " + ls));
+            ls.setBetaConstraintsDeriv(lambdaEqual, lambdaLessThan, _state, equalityConstraints, lessThanEqualToConstraints,
+                    ginfo, _state.beta());
+            return;
+          }
+
+          // update constraint parameters
+          updateConstraintParameters(_state, lambdaEqual, lambdaLessThan, equalityConstraints, lessThanEqualToConstraints);
+
+          // check for stopping conditions but will not stop for first iteration.
+          if (checkIterationDone(betaCnd, gradientInfo, iterCnt))
+            return;
+          firstIter = false;
+        }
+      } catch (NonSPDMatrixException e) {
+        Log.warn(LogMsg("Got Non SPD matrix, stopped."));
+      }
+    }
+*/
```

Comment:
If not used, please remove before merging.

Reply:
Removed.